recipes/git-annex/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e -o pipefail -x
+
+BINARY_HOME=$PREFIX/bin
+PACKAGE_HOME=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+
+mkdir -p $BINARY_HOME
+mkdir -p $PACKAGE_HOME
+
+export CPPFLAGS=-I${PREFIX}/include
+export CFLAGS=-I${PREFIX}/include
+export CXXFLAGS=-I${PREFIX}/include
+export CPPFLAGS=-I${PREFIX}/include
+export C_INCLUDE_PATH=${PREFIX}/include
+export CPLUS_INCLUDE_PATH=${PREFIX}/include
+export LDFLAGS=-L${PREFIX}/lib
+export LIBRARY_PATH=${PREFIX}/lib
```

Comment:
Do you need the env variables above? `toolchain` sets some of these.

Reply:
I've tried removing some of these, got errors -- see earlier commits.

Reply:
Then you should append instead of setting the env variables

Reply:
@isuruf this has been fixed.